tests/panda/test_panda.py

```diff
@@ -135,26 +136,9 @@ async def test_panda_block_missing_signals(pva):
 
 
 async def test_panda_unable_to_connect_to_pvi():
-    panda = PandA("pva://NON-EXISTENT")
+    panda = PandA("NON-EXISTENT:", "broken-panda")
 
     with pytest.raises(NotConnected) as exc:
         await panda.connect(timeout=0.01)
 
     assert exc.value._errors == "pva://NON-EXISTENT:PVI"
-
-    files = [
-        __file__,
-        "ophyd_async/panda/panda.py",
-        "ophyd_async/panda/panda.py",
-    ]
-    funcs = ["test_panda_unable_to_connect_to_pvi", "connect", "pvi"]
-    lines = [
-        "await panda.connect(timeout=0.01)",
-        'await pvi(self._init_prefix + ":PVI", self.ctxt, timeout=timeout)',
-        "raise NotConnected(pv) from exc",
-    ]
-
-    for idx, each_frame in enumerate(traceback.extract_tb(exc.tb)):
-        assert each_frame.filename.endswith(files[idx])
-        assert each_frame.name == funcs[idx]
-        assert each_frame.line == lines[idx]
```

Comment:
Was this level of where the exception is being thrown actually useful? Since this has been refactored from line 2 downwards, I'm only putting it back if there's serious outcry.

Reply:
Can be removed